spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/StringQuery.java

```diff
@@ -68,6 +67,7 @@ class StringQuery implements DeclaredQuery {
 	 *
 	 * @param query must not be {@literal null} or empty.
 	 */
+	@Deprecated
```

Comment:
Why is this deprecated? This leaves us without a deprecated constructor.

Reply:
I think this is relic from when I tried to introduce "modes". I'll remove it.